src/project.sh

```
@@ -1,31 +1,35 @@
     IFS=':' read -r workspace_path workspace_type <<< "$workspace"
-
+    if [ -z "$workspace_path" ]; then
+      do_error "No workspace path provided"
+    fi
+    if [ -z "$workspace_type" ]; then
+      do_error "No workspace type provided"
+    fi
     if [ -z "$main_workspace_path" ]; then
       main_workspace_path=$(realpath "$workspace_path")
     else
       cd "$main_workspace_path/$workspace_path"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)